platform/platform-app/src/main/resources/features.xml

```diff
@@ -834,6 +834,7 @@
 
     <feature name ="platform-migration" install="manual" version="${project.version}"
              description="Provides ability to export configurations.">
```

Comment:
:question: Will there ever be a case where a PersistenceStrategy is added to the system that is not in platform-io-impl?  If so, it seems like this would not work.  Is it bad to fail here (https://github.com/codice/ddf/blob/master/platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigratable.java#L60-L62) since osgi services are meant to be dynamic?

Reply:
Feature files need to be maintained as implementations change because they specify how we wire up the container. While adding a new strategy to `platform-io-impl` is preferable, external strategies will also need to become dependencies of the consuming feature.

Reply:
@figliold great question. I discussed with @Lambeaux and @lessarderic. This change is okay. It seems like the default persistence strategy is okay to define, as in platform should provide a default one. In general, feature files can be updated as new implementations of services become available.